ovos_cli_client/text_client.py

```diff
@@ -22,8 +22,9 @@
 import textwrap
 import time
 from threading import Thread, Lock
-
 from math import ceil
+
+import mycroft
```

Comment:
this module is supposed to independent of mycroft, if this import is needed then the correct step is to refactor the place where it is used.... i believe it's just for a version string, so maybe wrap that in a try: except: ?

Reply:
since this is fixed on the dev branch and only on a problem on master (pip install points to master) this is only temporary solution anyway until this gets the usual versioning.